src/pytorch_lightning/lite/lite.py

```diff
@@ -11,39 +11,58 @@
+from lightning_lite.plugins import CheckpointIO, ClusterEnvironment
+from lightning_lite.plugins import DeepSpeedPrecision as LiteDeepSpeedPrecision
+from lightning_lite.plugins import DoublePrecision as LiteDoublePrecision
+from lightning_lite.plugins import NativeMixedPrecision as LiteNativeMixedPrecision
+from lightning_lite.plugins import Precision as LitePrecision
+from lightning_lite.plugins import TPUBf16Precision as LiteTPUBf16Precision
+from lightning_lite.plugins import TPUPrecision as LiteTPUPrecision
+from lightning_lite.strategies import DataParallelStrategy as LiteDataParallelStrategy
+from lightning_lite.strategies import DDPShardedStrategy as LiteDDPShardedStrategy
+from lightning_lite.strategies import DDPSpawnShardedStrategy as LiteDDPSpawnShardedStrategy
+from lightning_lite.strategies import DDPSpawnStrategy as LiteDDPSpawnStrategy
+from lightning_lite.strategies import DDPStrategy as LiteDDPStrategy
+from lightning_lite.strategies import DeepSpeedStrategy as LiteDeepSpeedStrategy
+from lightning_lite.strategies import SingleDeviceStrategy as LiteSingleDeviceStrategy
+from lightning_lite.strategies import SingleTPUStrategy as LiteSingleTPUStrategy
+from lightning_lite.strategies import Strategy as LiteStrategy
+from lightning_lite.strategies import XLAStrategy
+from pytorch_lightning.accelerators import Accelerator as PLAccelerator
+from pytorch_lightning.plugins import CheckpointIO as PLCheckpointIO
+from pytorch_lightning.plugins import ClusterEnvironment as PLClusterEnvironment
+from pytorch_lightning.plugins import DeepSpeedPrecisionPlugin as PLDeepSpeedPrecisionPlugin
+from pytorch_lightning.plugins import DoublePrecisionPlugin as PLDoublePrecisionPlugin
+from pytorch_lightning.plugins import NativeMixedPrecisionPlugin as PLNativeMixedPrecisionPlugin
+from pytorch_lightning.plugins import PrecisionPlugin as PLPrecisionPlugin
+from pytorch_lightning.plugins import TPUBf16PrecisionPlugin as PLTPUBf16PrecisionPlugin
+from pytorch_lightning.plugins import TPUPrecisionPlugin as PLTPUPrecisionPlugin
+from pytorch_lightning.strategies import DataParallelStrategy as PLDataParallelStrategy
+from pytorch_lightning.strategies import DDPShardedStrategy as PLDDPShardedStrategy
+from pytorch_lightning.strategies import DDPSpawnShardedStrategy as PLDDPSpawnShardedStrategy
+from pytorch_lightning.strategies import DDPSpawnStrategy as PLDDPSpawnStrategy
+from pytorch_lightning.strategies import DDPStrategy as PLDDPStrategy
+from pytorch_lightning.strategies import DeepSpeedStrategy as PLDeepSpeedStrategy
+from pytorch_lightning.strategies import SingleDeviceStrategy as PLSingleDeviceStrategy
+from pytorch_lightning.strategies import SingleTPUStrategy as PLSingleTPUStrategy
+from pytorch_lightning.strategies import Strategy as PLStrategy
+from pytorch_lightning.strategies import TPUSpawnStrategy as PLTPUSpawnStrategy
+
+_PluginsTuple = (PLPrecisionPlugin, PLClusterEnvironment, PLCheckpointIO)
```

Comment:
Why not?:

```suggestion
_PL_PLUGIN_TUPLE = (PLPrecisionPlugin, PLClusterEnvironment, PLCheckpointIO)
```

Reply:
It's not actually used. I don't remember why I added it. 
Removed